src/FormControlFeedback.js

```diff
@@ -0,0 +1,64 @@
+import classNames from 'classnames';
+import React from 'react';
+
+import bootstrapUtils, { bsClass } from './utils/bootstrapUtils';
+
+import Glyphicon from './Glyphicon';
+
+const defaultProps = {
+  bsRole: 'feedback',
+};
+
+const contextTypes = {
+  $bs_formGroup: React.PropTypes.object,
+};
+
+class FormControlFeedback extends React.Component {
+  getGlyph(validationState) {
+    switch (validationState) {
+    case 'success': return 'ok';
+    case 'warning': return 'warning-sign';
+    case 'error': return 'remove';
+    default: return null;
```

Comment:
I think the icons used are just "suggested" or examples, I don't think there is any reason why they need to be these three. perhaps  `this.props.glyph` should be used used if it exists?